### ISSUE TITLE
invalid static and templates path

### ISSUE THREAD

**Issue by Akilan1999** [uadmin/uadmin#85]
When I run `uadmin prepare`, I get:
```
[ WARNING]   Unable to copy "static" folder: lstat /home/akilan/go/pkg/mod/github.com/uadmin/uadmin@vv0.6.2/static: no such file or directory
```

**Comment by Spotlight64** [comment]
There are two ways to resolve the issue:
1. You can perform a command in your terminal with **uadmin prepare --src**.
2. If solution 1 doesn't work, you have to remove the extra v in **go.mod** file.

```
 ... uadmin/uadmin@vv0.6.2/static ...
```

To

```
 ... uadmin/uadmin@v0.6.2/static ...
```

**Comment by maitredede** [comment]
Hello,
Same problem, but path is different :
```
[ WARNING]   Unable to copy "static" folder: lstat /go/pkg/mod/github.com/uadmin/uadmin@v0.8.8/static: no such file or directory
```
I am using golang 1.18, and I installed uadmin tool with : `go install github.com/uadmin/uadmin/cmd/uadmin@latest`. Installed version is `v0.8.3`

**Comment by TwistedHardware** [comment]
Fixed in v0.8.0
